### PR TITLE
Updates to 'quick start' section

### PR DESCRIPTION
**Pull request by Ela-valorem**
As a new user of JavaScript, Angular, and Azure AD within the context 
of an ASP.NET WebAPI application, I found it challenging to get started 
with the library.

Hoping to help others who are beginners, I have added to the existing 
Quick Start section to make it more visible, more thoroughly explain 
the beginner steps, include links to complementary information 
in context, and expand on the differences between single and 
multi-tenant setup.

### CODE REVIEW

**Comment by tushargupta51** [comment]
Thanks @Ela-valorem for the PR. We will review it now.

File: README.md [hunk 1]
```diff
@@ -141,10 +152,13 @@ $routeProvider.
...
 
 ***Optional***
-7- If you so choose, in addition (or substitution) to route level protection you can add explicit login/logout UX elements. Furthermore, you can access properties of the currently signed in user directly form JavaScript (via userInfo and userInfo.profile):
+7- If you so choose, in addition (or substitution) to route level protection you can add explicit login/logout UX elements. Furthermore, you can access properties of the currently signed in user directly form JavaScript (via userInfo and userInfo.profile).
+The userInfo.profile property provides access to the authentication token received from AAD. The AAD token contains claims that can be used by the application for validation, to identify the subject's directory tenant, and so on. The complete list of claims with a brief description of each value is here, [Claims in Azure AD Security Tokens](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-scenarios%23claims-in-azure-ad-security-tokens):
```
Comment (tushargupta51) [inline README.md]: Please rephrase this to:

The userInfo.profile property provides access to the claims in the id token received from AAD. The claims can be used by the application for validation, to identify the subject's directory tenant, and so on. The list of claims with a brief description of each value is here, [Claims in Azure AD Security Tokens](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-authentication-scenarios#claims-in-azure-ad-security-tokens)

File: README.md [hunk 2]
```diff
@@ -141,10 +152,13 @@ $routeProvider.
     });
 
 ```
-6- Any service invocation code you might have will remain unchanged. Adal's interceptor will automatically add tokens for every outgoing call.
+6- Any service invocation code you might have will remain unchanged. ADAL's interceptor will automatically add tokens for every outgoing call. 
+
+Anonymous endpoints, introduced in version 1.0.10, is an array of values that will be ignored by the ADAL route changed handler. ADAL will not attach a token to outgoing requests that have these keywords or URI. Routes that *do not* specify the ```requireADLogin=true``` property are added to the ```anonymousEndpoints``` array automatically.
```
Comment (tushargupta51) [inline README.md]: route changed handler => route/state change handlers

File: README.md [hunk 3]
```diff
@@ -12,7 +12,9 @@ You can find the changes for each version in the [change log](https://github.com
 
 ## Samples and Documentation
 
-[We provide a full suite of sample applications and documentation on GitHub](https://github.com/azure-samples?query=active-directory) to help you get started with learning the Azure Identity system. This includes tutorials for native clients such as Windows, Windows Phone, iOS, OSX, Android, and Linux. We also provide full walkthroughs for authentication flows such as OAuth2, OpenID Connect, Graph API, and other awesome features. 
+[We provide a full suite of sample applications and documentation on GitHub](https://github.com/azure-samples?query=active-directory) to help you get started with learning the Azure Identity system. This includes tutorials for native clients such as Windows, Windows Phone, iOS, OSX, Android, and Linux. We also provide full walkthroughs for authentication flows such as OAuth2, OpenID Connect, Graph API, and other awesome features.
+ 
+Instructions for setting up an Azure AD application are here, [Azure Active Directory authentication in Azure Portal](https://docs.microsoft.com/en-us/azure/app-service-mobile/app-service-mobile-how-to-configure-active-directory-authentication).
```
Comment (tushargupta51) [inline README.md]: I am not sure if that is the general link to configure applications in AAD portal. I would prefer to skip these instructions here. Our samples also have a detailed guide to register an application with AAD.

Comment (Ela-valorem) [inline README.md]: I think saying that additional information is available in the samples would be good enough. I thought providing the link here would tie the two systems together more clearly. Thus, it doesn't matter where you start investigating Azure AD, you can get to the documentation you need to get started building an app.

File: README.md (hunk 2 above)
Comment (Ela-valorem) [inline README.md]: Will do

File: README.md (hunk 1 above)
Comment (Ela-valorem) [inline README.md]: Will do.

File: README.md (hunk 3 above)
Comment (tushargupta51) [inline README.md]: @dstrockis / @vibronet  What do you think about adding this info here? Is that correct link to use here?

**Comment by Ela-valorem** [comment]
The requested updates have been completed. Please let me know if there's anything else.

**Comment by tushargupta51** [comment]
@Ela-valorem Thanks for your contribution! The changes look good to me, I will merge the PR.
